src/ui.tsx

```diff
@@ -138,45 +164,77 @@ export class UserInterface extends Component<Props, State> {
       });
   };
 
-  render = (): ReactNode => (
-    <div style={{ flexGrow: 1 }}>
-      <AppBar position="static">
-        <Toolbar>
-          <Typography variant="h6">CURATE</Typography>
-          <ComboBox
-            metadata={this.state.metadata}
-            metadataKeys={this.state.metadataKeys}
-            callback={this.handleOnSearchSubmit}
-          />
-        </Toolbar>
-      </AppBar>
-
-      <Grid container spacing={0} wrap="nowrap">
-        <Grid item style={{ position: "relative", width: "200px" }}>
+  render = (): ReactNode => {
+    const { classes } = this.props;
+    return (
+      <div className={classes.root}>
+        <CssBaseline />
+        <AppBar position="fixed" className={classes.appBar}>
+          <Toolbar>
+            <Typography variant="h6">CURATE</Typography>
+            <ComboBox
+              metadata={this.state.metadata}
+              metadataKeys={this.state.metadataKeys}
+              callback={this.handleOnSearchSubmit}
+            />
+          </Toolbar>
+        </AppBar>
+
+        <div>
+          <Toolbar />
+          {/* empty Toolbar element pushes the next element down by the same width as the appbar, preventing it rendering behind the appbar when position="fixed" (see https://material-ui.com/components/app-bar/#fixed-placement) */}
           <LabelsAccordion
             expanded={this.state.expanded === "labels-toolbox"}
             handleToolboxChange={this.handleToolboxChange("labels-toolbox")}
             imageLabels={this.state.imageLabels}
             callback={this.handleOnLabelSelection}
           />
-        </Grid>
-        <Grid item style={{ position: "relative", width: "80%" }}>
-          <Grid container spacing={3}>
+        </div>
+
+        <Grid
+          container
+          style={{ position: "relative", width: "80%", margin: "16px" }}
+        >
+          <Toolbar />
+          {/* empty Toolbar element pushes the next element down by the same width as the appbar, preventing it rendering behind the appbar when position="fixed" (see https://material-ui.com/components/app-bar/#fixed-placement) */}
+          <Grid container spacing={3} wrap="wrap">
             {this.props.tiles.map(
-              (tile) =>
+              (tile, index) =>
                 this.isTileInSelectedImages(tile.name) && (
-                  <Grid item xs={1} key={tile.id}>
+                  <Grid
+                    item
+                    key={tile.id}
+                    style={{
+                      backgroundColor:
+                        this.state.selected === index && "lightblue",
+                    }}
+                  >
                     <img
                       height={128}
                       src={`data:image/png;base64,${tile.thumbnail}`}
                       alt={tile.name}
+                      onClick={() => {
```

Comment:
The solution to this is probably wrap the image in an `a` tag?

Reply:
Perhaps that is the answer, but I went with `Button` before I saw this and it turned out to look cooler than I expected.

Reply:
yea, that's probably the correct way anyway!